web/blog/vast-v3.0/index.md

```diff
@@ -0,0 +1,380 @@
+---
+draft: true
+title: VAST v3.0
+description: VASTQL Refreshments
+authors: [dominiklohmann, mavam]
+date: 2023-01-20
+image: /path/to/image.png
+tags: [release, vastql, cef, performance, introspection, regex]
+---
+
+[VAST v3.0][github-vast-release] is out. The release brings some major updates
+to VAST's query language, including the ability to search using regular
+expressions.
+
+[github-vast-release]: https://github.com/tenzir/vast/releases/tag/v3.0.0
+
+<!--truncate-->
+
+## VASTQL Upgrades
```

Comment:
This needs another section now that we merged the initial version of export pipelines. We should also mention the head and taste operators, and the deprecation of the operator YAML syntax.